extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerSketchFetcher.java

```diff
@@ -118,22 +140,17 @@ CompletableFuture<Either<Long, ClusterByPartitions>> inMemoryFullSketchMerging(
     final int workerCount = workerTaskIds.size();
     // Guarded by synchronized mergedStatisticsCollector
     final Set<Integer> finishedWorkers = new HashSet<>();
+    final Queue<Future<?>> executorFutures = new ConcurrentLinkedQueue<>();
```

Comment:
is there a reason for making this `ConcurrentLinkedQueue` instead of a simple list?

Reply:
Discussed offline. Changed to a simple list as this would be enough